app/forms/green_lanes/applicable_exemptions_form.rb

```diff
@@ -46,5 +54,28 @@ def presented_answers
         acc[key.split('_').last] = value.select(&:present?)
       end
     end
+
+    private
+
+    # Output: A list of exemptions (conditions) that appear more than once in the form,
+    #         where the user has selected only one of the occurrences.
+    def ambiguous_exemptions
+      answered_exemptions = answers.values.flatten.reject { |e| ['none', ''].include?(e) }
+
+      count_answers = answered_exemptions.group_by { |code| code }
+                                         .transform_values(&:count)
+
+      count_answers.reject { |code, count| count == exemptions_count[code] }
+                   .keys
+    end
+
+    def exemptions_count
+      all_exemptions = category_assessments.flat_map do |ca|
+        ca.exemptions.map(&:resource_id)
+      end
+
+      all_exemptions.group_by { |code| code }
+                    .transform_values(&:count)
+    end
   end
```

Comment:
This is the core of this PR.
What it does is to count the checked answers,
and count the exemptions available,
then they are compared.

If they do not match, a validation error will be shown to the user.